consensus/istanbul/backend/engine.go

```diff
@@ -271,22 +293,16 @@ func (sb *backend) verifyCommittedSeals(chain consensus.ChainReader, header *typ
 	validators := snap.ValSet.Copy()
 	// Check whether the committed seals are generated by parent's validators
 	validSeal := 0
-	proposalSeal := istanbulCore.PrepareCommittedSeal(header.Hash())
-	// 1. Get committed seals from current header
-	for _, seal := range extra.CommittedSeal {
-		// 2. Get the original address by seal and parent block hash
-		addr, err := istanbul.GetSignatureAddress(proposalSeal, seal)
-		if err != nil {
-			sb.logger.Error("not a valid address", "err", err)
-			return errInvalidSignature
-		}
-		// Every validator can have only one seal. If more than one seals are signed by a
-		// validator, the validator cannot be found and errInvalidCommittedSeals is returned.
+	committers, err := sb.Signers(header)
+	if err != nil {
+		return errInvalidSignature
```

Comment:
Instead of returning `errInvalidSignature`, can we just return `err` as `Signers()` can return other error type